src/OpenLoco/src/ConfigConvert.hpp

```diff
@@ -123,6 +124,34 @@ namespace YAML
         }
     };
 
+    // OpenLoco::ObjectHeader
+    template<>
+    struct convert<OpenLoco::ObjectHeader>
+    {
+        static Node encode(const OpenLoco::ObjectHeader& rhs)
+        {
+            Node node;
+            node["flags"] = rhs.flags;
+            node["name"] = std::string(rhs.name, 8);
+            node["checksum"] = rhs.checksum;
+            return node;
+        }
+
+        static bool decode(const Node& node, OpenLoco::ObjectHeader& rhs)
+        {
+            if (node.IsMap())
+            {
+                rhs.flags = node["flags"].as<uint32_t>(0);
```

Comment:
Should probably default to kEmptyObjectHeader. Which is all of the fields set to 0xFF including the name. So `0xFFFFFFFFU`, `"\xFF\xFF\xFF\xFF\xFF\xFF\xFF\xFF"`, `0xFFFFFFFFU`

Reply:
I suppose I should just copy them from `kEmptyObjectHeader` instead, then, right?

Reply:
sure.